### PR TITLE
Update documentation to refer new contributors to `difficulty:easy` labeled issues

### PR DESCRIPTION
**Pull request by wldcordeiro**
Associated Issue: #877 

### Summary of Changes

* remove references to the defunct `first-timers-only` label in favor of the `difficulty:easy` label

### Testing

* [x] passes `npm test`
* [x] passes `npm run lint`

### CODE REVIEW

**Comment by jasonLaster** [comment]
Hmm, I think we should say look at up for grabs, we have easy medium and hard. Some people like Thomas jump right into the hard ones!

**Comment by wldcordeiro** [comment]
Maybe phrase it around `difficulty:easy` being good starting points but to find what interests you that's `up-for-grabs`? I'll play with the wording a bit.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -167,11 +167,15 @@ If you find an `up for grabs` issue without a difficulty level set as a label or
 
 #### Your First Code Contribution
 
-If this is your first time contributing any code to an open source project take a look through the `first-timers-only` issues:
+A good place to start when looking for something to contribute, is looking through the `up-for-grabs` issues. These are actionable issues that no one has laid claim to.
```
Comment (jasonLaster) [inline CONTRIBUTING.md]: Hmm, maybe something a little more active

```
If you're looking for a good issue, you can look through the `up-for-grabs` issues. These issues should be actionable and well documented.

There are several difficulty levels, *easy*, *medium*, *hard*. We recommend grabbing an *easy* issue, but it's up to you.

* [up-for-grabs][labels-up-for-grabs] - issues that are not assigned to anyone and are available to be worked on.
* [difficulty:easy][labels-difficulty-easy] - issues which should have clear expectations and a mentor to help you through
  		  
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -167,11 +167,15 @@ If you find an `up for grabs` issue without a difficulty level set as a label or
...
-If this is your first time contributing any code to an open source project take a look through the `first-timers-only` issues:
+If you're looking for a good issue, you can look through the `up-for-grabs` issues. These issues should be actionable and well documented.
 
-* [first-timers-only][labels-first-timers-only] - issues which should have clear expectations and a mentor to help you through
+There are several difficulty levels, *easy*, *medium*, *hard*. We recommend grabbing an *easy* issue, but it's up to you.
+
+* [up-for-grabs][labels-up-for-grabs] - issues that are not assigned to anyone and are available to be worked on.
+* [difficulty:easy][labels-difficulty-easy] - issues which should have clear expectations and a mentor to help you through.
+* [difficulty:medium][labels-difficulty-medium] - issues that are typically more complex than *easy* issues and may not have as clear expectations.
+* [difficulty:hard][labels-difficulty-hard] - issues which are complex, lacking expectations or touch lots of the codebase.
```
Comment (jasonLaster) [inline CONTRIBUTING.md]: hmm, maybe one more pass:

```
- clear expectations and a mentor to help you through
- more complex and may not have as clear expectations
- complex and has some open technical questions
```

File: public/js/components/Editor.css [hunk 3]
```diff
@@ -14,6 +14,8 @@
   width: 100%;
   top: 30px;
   left: 0px;
+  display: flex;
+  flex-direction: column;
 }
```
Comment (jasonLaster) [inline public/js/components/Editor.css]: haha - should this be here?

Comment (wldcordeiro) [inline public/js/components/Editor.css]: Oh noez, unstaged stuff got pulled in!

**Comment by jlongster** [comment]
The error above is the sourcemap timeout test which has been fixed on master.

**Pull request merged**